docs/tables/aws_clb_access_log/index.md

```diff
@@ -75,7 +77,9 @@ select
   timestamp,
   elb,
   tp_index as account_id,
-  request,
+  request_http_version,
+  request_http_method,
+  request_url,
```

Comment:
I propose we use same name for these fields as we approved for nginx/apache: 
- `request_uri`
- `request_method`
- `server_protocol`

@cbruno10 - agree?